docs/stock/understanding-highcharts-stock.md

```diff
@@ -5,30 +5,92 @@ Highcharts Stock is based on Highcharts, meaning it has all the core functionali
 
-Navigator
+<iframe style="width: 100%; height: 550px; border: none;" src=https://highcharts.github.io/highcharts-utils/samples/#gh/352ea24215ec4f79bb50ead0152430087bc3551e/sample/stock/interactive-docs/series-type allow="fullscreen"></iframe>
+
+```js
+chart.series[0].update({
+    type: 'candlestick'
+});
+```
+
+See [Update series method](https://api.highcharts.com/class-reference/Highcharts.Series#update) and [series API options](https://api.highcharts.com/highstock/series).
+
+Navigator and scrollbar
 ---------
 
-Allows you to fine tune the range of the chart which is displayed.
+Allows you to fine tune the range of the chart which is displayed and scroll through it.
+
+See [Navigator](https://highcharts.com/docs/stock/navigator) for more information.
+
+<iframe style="width: 100%; height: 640px; border: none;" src=https://highcharts.github.io/highcharts-utils/samples/#gh/352ea24215ec4f79bb50ead0152430087bc3551e/sample/stock/interactive-docs/navigator allow="fullscreen"></iframe>
+
+```js
+chart.update({
+    navigator: {
+        enabled: true,
+        height: 100
+    },
+    scrollbar: {
+        enabled: false
+    }
+});
+```
+To update navigator and scrollbar use [Chart update](https://api.highcharts.com/class-reference/Highcharts.Chart#update). See [navigator API options](https://api.highcharts.com/highstock/navigator).
 
-See [Navigator](https://highcharts.com/docs/stock/navigator) for more information.
 
 Range selector
 --------------
 
 Allows you to quickly select a range to be shown on the chart or specify the exact interval to be shown.
 
+<iframe style="width: 100%; height: 600px; border: none;" src=https://highcharts.github.io/highcharts-utils/samples/#gh/352ea24215ec4f79bb50ead0152430087bc3551e/sample/stock/interactive-docs/range-selector allow="fullscreen"></iframe>
+
 See [Range selector](https://highcharts.com/docs/stock/range-selector) for more information.
 
-Scrollbar
+```js
+chart.update({
+    rangeSelector: {
+        enabled: true,
+    }
+});
+```
+
+To update range selector use [Chart update](https://api.highcharts.com/class-reference/Highcharts.Chart#update). See [range selector API options](https://api.highcharts.com/highstock/rangeSelector).
+
+Crosshair
 ---------
 
-Allows scrolling on the chart.
+Shows a line perpendicular to the corresponding axis which is following the mouse position or nearest point, depending on the `crosshair.snap` property. This functionality can be found in the [axis API](https://api.highcharts.com/highstock/xAxis.crosshair) options. Crosshairs can also be used in Highcharts Core (without Stock module), but are not enabled by default.
 
-See [Scrollbar](https://highcharts.com/docs/chart-concepts/scrollbar) for more information.
+<iframe style="width: 100%; height: 600px; border: none;" src=https://highcharts.github.io/highcharts-utils/samples/#gh/352ea24215ec4f79bb50ead0152430087bc3551e/sample/stock/interactive-docs/crosshair allow="fullscreen"></iframe>
```

Comment:
Could we add one more button? With enable/disable action for the label?

Reply:
Added in #21537